### ISSUE TITLE
Daemon loses track of config file

### ISSUE THREAD

**Issue by direc85** [direc85/harbour-batterybuddy#12]
PawelSpoon wrote:

> Hi, since the last update I get notifications on low battery, but I did not choose the option. So should not receive any notification interval is also ignored. Do get a message every minute.

nephros wrote:

```
[General]
highAlert=@ByteArray(85)
highLimit=@ByteArray(85)
interval=@ByteArray(600)
limitEnabled=@ByteArray(0)
lowAlert=@ByteArray(10)
lowLimit=@ByteArray(49)
notificationHighText=Please disconnect the charger.
notificationLowText=Please connect the charger.
notificationTitle=Battery charge %1%
notificationsEnabled=@ByteArray(0)
```

I suspect that the daemon file watcher loses track of the config file, and has no mechanism to recover. It's rather unpleasant to debug, since restarting the daemon fixes the issue.

**Comment by direc85** [comment]
Perhaps this is linked to issue #8? In both cases changes to config file don't seem to change behavior.

**Comment by direc85** [comment]
There may be something funny going on with the services. Please upgrade to 3.4-1 and see if it works.

**Comment by direc85** [comment]
Can I assume that the bug was fixed with 3.4-1?

**Comment by direc85** [comment]
Closing for now. Please open a new ticket against the newest version if the issue still persists.
